examples/1d/elixir_euler_density_wave.jl

```diff
@@ -0,0 +1,53 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+equations = CompressibleEulerEquations1D(1.4)
+
+initial_condition = initial_condition_density_wave
+
+surface_flux = flux_central
+solver = DGSEM(5, surface_flux)
+
+coordinates_min = -1
+coordinates_max =  1
+mesh = TreeMesh(coordinates_min, coordinates_max,
+                initial_refinement_level=2,
+                n_cells_max=30_000)
+
+
+semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition, solver)
+
+
+###############################################################################
+# ODE solvers, callbacks etc.
+
+tspan = (0.0, 2.0)
+ode = semidiscretize(semi, tspan)
+
+summary_callback = SummaryCallback()
+
+stepsize_callback = StepsizeCallback(cfl=0.8)
```

Comment:
```suggestion

# FIXME Taal restore after Taam sync
stepsize_callback = StepsizeCallback(cfl=0.8)
```

Reply:
I don't think there is anything to change, since this is 1D. Or am I missing something?

Reply:
I just wasn't sure whether there were any changes in the 1D implementation. Since you included this note in every other example, I thought we should also revisit this one later. If you're sure that nothing has to be changed you can ignore this one.

Reply:
Good point. This should be revisited in batch when the time step calculation is updated.